packages/components/src/__actions__/Menu/v3/Menu.module.scss

```diff
@@ -0,0 +1,38 @@
+@import "~@kaizen/design-tokens/sass/border";
+@import "~@kaizen/design-tokens/sass/color";
+@import "~@kaizen/design-tokens/sass/shadow";
+@import "~@kaizen/design-tokens/sass/spacing";
+@import "~@kaizen/design-tokens/sass/typography";
+
+.menu {
+  background-color: $color-white;
+  color: $color-purple-800;
+  width: 248px;
+  max-height: 22rem;
+  overflow: auto;
+  padding-block: $spacing-6;
+  outline: none;
+  border-radius: $border-solid-border-radius;
+  box-shadow: $shadow-large-box-shadow;
```

Comment:
Also try convert these to CSS vars

Reply:
@mcwinter07 you might also want to try this, as a inch towards CSS modules

Reply:
Happy to give it a crack 💥 🔨

Reply:
Done